packages/app/src/cli/services/app-logs/logs-command/ui/components/Logs.test.tsx

```diff
@@ -45,114 +45,6 @@ const NETWORK_ACCESS_HTTP_RESPONSE = {
   },
 }
 
-const USE_POLL_APP_LOGS_RETURN_VALUE = {
-  appLogOutputs: [
```

Comment:
Lot of changes here, but I refactored this to test each output individually. This should be easier to add tests for specific output cases in the future.